### DISCUSSION TITLE
how to migrate from storybook 6.5 to 7.0

### DISCUSSION THREAD

**Discussion by MoldyLunchBox** [storybookjs/storybook#26999] [Help]
### Summary

i tried `npx storybook@7.0 upgrade`  but for some reason it keeps going for the latest version 

any help or remarks are greatly appreciated

` ``> npx storybook@7.0 upgrade   
 • Checking for latest versions of '@storybook/*' packagesinfo ,Upgrading  ***\migration\package.json
info
info  @storybook/addon-actions       ^6.5.14  →   ^8.0.9
info  @storybook/addon-essentials    ^6.5.14  →   ^8.0.9
info  @storybook/addon-interactions  ^6.5.14  →   ^8.0.9
info  @storybook/addon-links         ^6.5.14  →   ^8.0.9
info  @storybook/builder-webpack5    ^6.5.15  →   ^8.0.9
info  @storybook/manager-webpack5    ^6.5.15  →  ^6.5.16
info  @storybook/react               ^6.5.14  →   ^8.0.9
info  @storybook/testing-library     ^0.0.13  →   ^0.2.2```

**Comment by valentinpalkovic** [comment]
Version 7.0 was only capable to always upgrade to the latest version. Please try to use the latest 7 version and then it should work, because the upgrade command was fixed in 7.6.x to be able to upgrade to a particular version.

`npx storybook@7.6.18 upgrade`.

Can you tell us why an upgrade to 8.0 does not work for you?

**Comment by MoldyLunchBox** [comment]
thanks a lot for replying.  could you tell me how you found this out? i spent two days searching

the reason we choose to migrate from 6.5 to 7 is because migrating to 8 will break a lot more  things.  i imagine it's going to be so broken to the point where it might make sense to start from scratch than deal with searching and patching all the broken things. but this is just an opinion. what do you think
